components/brave_rewards/resources/extension/brave_rewards/components/app.tsx

```diff
@@ -25,17 +26,25 @@ interface Props extends RewardsExtension.ComponentProps {
 
 interface State {
   windowId: number
+  onlyAnonWallet: boolean
 }
 
 export class RewardsPanel extends React.Component<Props, State> {
   constructor (props: Props) {
     super(props)
     this.state = {
-      windowId: -1
+      windowId: -1,
+      onlyAnonWallet: false
     }
   }
 
   componentDidMount () {
+    chrome.braveRewards.onlyAnonWallet((only: boolean) => {
+      this.setState({
+        onlyAnonWallet: !!only
```

Comment:
Forgive my lack of typescript knowledge in advance, but this !! doesn't look right to me so wanted to point it out in case it is a typo :)

Reply:
it's ok. This just ensures if for some reason variable is undefined it will convert it to boolean. https://medium.com/better-programming/javascript-bang-bang-i-shot-you-down-use-of-double-bangs-in-javascript-7c9d94446054